README.md

```diff
@@ -29,6 +29,24 @@ docker-compose exec web /bin/sh -c "bundle exec rails db:setup"
 
 Then open http://localhost:3000 to see the app.
 
+## Setting up the app for local (non-Docker) development
+
+### Creating the DB user
+
+We use a project-specific Postgres user which needs to be created before you create and use the databases.
+
+On OSX:
+
+```bash
+psql -U postgres < db/create_dev_user.sql
```

Comment:
Why do we need this? Surely `rake db:setup` is enough?

Reply:
You will have a default user on your OS, why create a new one?

Reply:
Added user creating/dropping as part of the usual rake tasks. @dankmitchell this should address your concerns about non-standard setup.